OracleHTTPServer/dockerfiles/14.1.2.0.0/Dockerfile

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) 2025 Oracle and/or its affiliates. All rights reserved.
+
+
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# ORACLE HTTP Server DOCKERFILES PROJECT
+# -----------------------------------------
+# This is the DockerFile for Oracle HTTP Server 14.1.2.0.0
+#
+# IMPORTANT
+# -------------------------------------------
+# The resulting image of this Dockerfile would have Oracle Linux 8 and 9 and Oracle HTTP Server 14.1.2.0.0 installed in standalone mode.
+
+# A standalone domain will be configured in this image
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) V1045136-01.zip Oracle HTTP Server 14.1.2.0 for Linux x86-64 installer
+#     Download from http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html
+#     unzip you will need the OHS installer (bin file) fmw_14.1.2.0.0_ohs_linux64.bin
+#
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# As root user run following command manually:
+#
+#     $ sh buildDockerImage.sh -v 14.1.2.0.0
+#
+# Alternatively you can also run
+# 
+#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/$IMAGE_NAME:$VERSION-$INSTALL_TYPE .
+#
+# For example
+# 
+#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/ohs:14.1.2.0.0 .
+#
+# Pull serverjre:8 base image
```

Comment:
Should this line and line 42 use the same image name?

Reply:
Yes